pandas/tests/extension/test_integer.py

```diff
@@ -20,6 +20,7 @@
 
 import pandas as pd
 import pandas._testing as tm
+from pandas.api.types import is_integer_dtype
```

Comment:
can you change the import on L19 as well (that is an incorrect location as well)

Reply:
@jreback done